### PR TITLE
chore: document unreachable or untested paths in near-membrane-base/src/blue

### PR DESCRIPTION
**Pull request by rwaldron**
image

### CODE REVIEW

File: packages/near-membrane-base/src/blue.ts [hunk 1]
```diff
@@ -57,10 +57,12 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
     if (typeof target === 'function') {
         // this new shadow target function is never invoked just needed to anchor the realm
         try {
+            // istanbul ignore next
             // eslint-disable-next-line func-names
             shadowTarget = 'prototype' in target ? function () {} : () => {};
```
Comment (jdalton) [inline packages/near-membrane-base/src/blue.ts:62]: ☝️ What's the reason to skip this one?

File: packages/near-membrane-base/src/blue.ts [hunk 2]
```diff
@@ -57,10 +57,12 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
...
         } catch {
             // target is a revoked proxy
+            // istanbul ignore next
             // eslint-disable-next-line func-names
             shadowTarget = function () {};
```
Comment (jdalton) [inline packages/near-membrane-base/src/blue.ts:67]: ☝️ And this one.

**Comment by jdalton** [comment]
Is the idea to come back and add the tests?

File: packages/near-membrane-base/src/blue.ts (hunk 1 above)
Comment (rwaldron) [reply packages/near-membrane-base/src/blue.ts:62]: I didn't add any additional comments here because it was already written on line 58 :D

**Comment by rwaldron** [comment]
> Is the idea to come back and add the tests?

Yes, or remove dead code.

File: packages/near-membrane-base/src/blue.ts (hunk 2 above)
Comment (rwaldron) [reply packages/near-membrane-base/src/blue.ts:67]: Same line 58 comment, but I think I want to add the explicit comments now.

**Review by jdalton — APPROVED** [review]

**Review by garychangsf — COMMENTED** [review]
I am not familiar with our team's code coverage principals. But do we plan to increase coverage for the ignored code paths? If so, why are we marking them as ignored?

**Comment by rwaldron** [comment]
@garychangsf 

> But do we plan to increase coverage for the ignored code paths? 

Yessir. My goal here was to bring us to 100% so we can more easily spot regressions moving forward, while also having a clear way to see what code needs tests. 👍

**Comment by jdalton** [comment]
Ah I see ok. We need to (as part of trust work then) be aware of the code coverage ignore comments and know to search them out to focus on those areas. Something to be mindful of when we have those discussions.

**Comment by rwaldron** [comment]
@jdalton that's my next step

**Comment by rwaldron** [comment]
It's also important to acknowledge that some places aren't reachable—and even say so in the comments. We need to decide if something is dead code, or something we want to keep but ignore.

**Pull request merged**